readme.rst

```diff
@@ -0,0 +1,46 @@
+Sentinel-5P Downloader
+======================
+
+.. image:: https://travis-ci.com/emissions-api/sentinel5dl.svg?branch=master
+    :target: https://travis-ci.com/emissions-api/sentinel5dl
+.. image:: https://img.shields.io/github/issues-raw/emissions-api/sentinel5dl?color=blue
+    :target: https://github.com/emissions-api/sentinel5dl/issues
+    :alt: GitHub issues
+.. image:: https://img.shields.io/github/license/emissions-api/sentinel5dl
+    :target: https://github.com/emissions-api/sentinel5dl/blob/master/LICENSE
+    :alt: MIT license
+
+This library provides easy access to data from the European Space Agency's
+Sentinel 5P satellite.
+
+- `Sentinel5dl on PyPI <https://pypi.org/project/sentinel5dl/>`_
+- `Online documentation <https://sentinel5dl.emissions-api.org>`_
```

Comment:
Why this link, here? Isn't this a reference on this page?

Reply:
You do not only show this on sentinel5dl.emissions-api.org but also on GitHub and soon on PyPI.